packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -89,4 +89,13 @@ export const keys = {
   ENTER: 'Enter',
   ARROWUP: 'ArrowUp',
   ARROWDOWN: 'ArrowDown',
-};
\ No newline at end of file
+};
+
+export function getItemfromLocalStorage(key) {
+  return JSON.parse(localStorage.getItem(key)) ? JSON.parse(localStorage.getItem(key)) : [];
```

Comment:
If the value doesn' exist i thing `localStorage.getItem(key)` returns the string "undefined" which JSON.parse doesn't handle well.

```suggestion
    const value = localStorage.getItem(key);
    return value !== 'undefined' ? JSON.parse(value) : null;
```

Reply:
Also some unit tests for these two would be good so we know how it handles non-successful retrievals and such.